webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateComponents/MonthView.js

```diff
@@ -21,6 +21,8 @@ class MonthView extends React.Component {
   componentDidUpdate = prevProps => {
     const newDate = this.props.date;
     if (prevProps.date !== newDate) {
+      // TODO: Fix #27114 - stop violating react/no-did-update-set-state
+      // eslint-disable-next-line react/no-did-update-set-state
```

Comment:
This PR does not aim to solve this issue.
https://projects.theforeman.org/issues/27114